BackEnd/src/business/season.ts

```diff
@@ -10,3 +11,11 @@ export async function CreateSeason(seasonName: string, providerId: number): Prom
 export async function GetSeasons(active: boolean): Promise<SeasonModel[]> {
   return await GetDbSeasons(active);
 }
+
+export async function GetSeasonBySearchName(searchName: string): Promise<SeasonModel> {
```

Comment:
Can we get rid of this?

Reply:
No, we need it for the frontend since we use the searchName for the URL
http://localhost:3000/leagues/summer2023unstoppable